index.html

```diff
@@ -0,0 +1,408 @@
+                        <div class="col-11">
+                            <h1 class="main_header">African women in tech group
+                                <br> launches to support diversity</h1>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-11 main_paragraph">
+                            <p>
+                                There is a significant gap of representation of women in the technology field in Africa, and having a support group in place
+                                that focuses on African women can be very beneficial to fill this gap.
+                            </p>
+                            <p>This group can also be a great platform to empower and inspire young women to pursue careers
+                                in technology and to become role models, by highlighting the successes and contributions
+                                of African women in the field, and providing access to mentorship and networking opportunities
+                                with successful female tech leaders</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-1 desk-div"></div>
+                        <div class="col-7 bottom-header">
+                            <p>
+                                <span class="header-paragraph">2023.10.15(THU)-16(FRI)</span>
+                                <br> @African women in tech group launches to support diversity and inclusion.</p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </main>
+
+        <div>
+
+            <!--midder section-->
+            <section class="container-fluid midder_section">
+                <div class="row">
+                    <div class="col-12">
+
+                        <center>
+
+                            <h2>Activities</h2>
+                        </center>
+
+                    </div>
+
+                    <section class="container cards-container">
+                        <div class="row cards">
+
+                            <div class="col-2">
+                                <div>
+                                    <center>
+                                        <img src="images/program_icon_01.png" alt="icon_one for the program">
+                                    </center>
+                                </div>
+                                <center>
+                                    <h3>Virtual events</h3>
+                                </center>
+                                <p>Many organizations and companies in Africa's technology sector have been turning to virtual
+                                    events and conferences.</p>
+                            </div>
```

Comment:
- [Optional] kindly articles those card listed in the `article` tag because it  a list of activities that can be like an article or semantic reason 👌